db/migrate/20180924172946_devise_create_users.rb

```diff
@@ -22,17 +20,16 @@ def change
       # t.inet     :last_sign_in_ip
 
       ## Confirmable
-      # t.string   :confirmation_token
-      # t.datetime :confirmed_at
-      # t.datetime :confirmation_sent_at
-      # t.string   :unconfirmed_email # Only if using reconfirmable
+      t.string   :confirmation_token
```

Comment:
You did well with the other column in the migration, please do not modify this migration. Or maybe you have an issue with rebase?

Reply:
I don't remember changing that. I'll change it back.